django/contrib/admin/static/admin/css/vendor/select2/select2.css

```diff
@@ -0,0 +1,482 @@
+.select2-container {
```

Comment:
@MarkusH I'd gladly drop it. But it's done the same way in other widgets JS code. Tho to be fair, this is our own.
Well just decide something, it's not like we can't add it again later.